### PR TITLE
CLI command (agdash) for autogluon_dashboard

### PR DESCRIPTION
**Pull request by shreyash2106**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 python3 -m venv .venv_dash 
 source .venv_dash/bin/activate
 ``` 
 2. Install all necessary libraries by running: `pip3 install -r requirements.txt`
-3. run `python3 dashboard.py --dataset_file 'path_to_csv_file.csv' --aggreagated_file 'path_to_csv_file.csv'` and open the provided http link in your browser. 
-<br> For example, `python3 dashboard.py --dataset_file 'dev_data/all_data.csv' --aggreagated_file 'dev_data/autogluon.csv'`
+3. Install the `autogluon_dashboard` package by running: `pip3 install -e .`. You can now interact with the dashboard through the CLI command `agdash`. 
+4. To convert a Panel app to WebAssemly,  Panel provides a script that will convert the code in `app.py` into an HTML file and JS file. This can be done in one line of code as: `panel convert app.py --to pyodide-worker --out web_files/`
+The generated HTML and JS files will be found in the `web_files` folder. These can be uploaded to the desired hosting service (eg: GitHub Pages or AWS S3). 
+<br>Run `agdash --dataset_file 'path_to_csv_file.csv' --aggreagated_file 'path_to_csv_file.csv'`.
```
Comment (yinweisu) [inline README.md:13]: I thought the cli should do the conversion for the user? But it seems like they need to call convert then call your cli?

Comment (shreyash2106) [reply README.md:13]: No they don't need to call convert - I just gave an example on how to use panel convert. The user just has to run the agdash ... command

Comment (shreyash2106) [reply README.md:13]: If it's confusing I can remove it from the README or reword it.

**Review by yinweisu — APPROVED** [review]
LGTM

**Pull request merged**